packages/server/src/tokens.ts

```diff
@@ -2,13 +2,13 @@ import { Token } from '@lumino/coreutils';
 
 import { ISignal } from '@lumino/signaling';
 
-import SW_URL from '!!file-loader?name=[name]-[contenthash:7].[ext]&context=.!./service-worker';
+import SW_URL from './service-worker?text';
```

Comment:
The downstream users need to add the below config to webpack to use this variable.

```
        {
          resourceQuery: /text/,
          type: 'asset/resource',
          generator: {
            filename: '[name]-[contenthash:7][ext]',
          },
        },
```

Maybe we should add this to the migration guide.

Reply:
Makes sense, we can add it to the migration guide page: https://jupyterlite.readthedocs.io/en/latest/migration.html

Reply:
To simplify downstream usage, we can export these specific Webpack configs so other apps can just extend their configs with it.